### DISCUSSION TITLE
Bug on attachment fields while importing CSV

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#9043] [Help] [answered]
If I try to import a CSV file into my bb table, the attachment field is not properly mapped and the final result is funny...
<img width="949" alt="2022-12-14 11_10_50-" src="https://user-images.githubusercontent.com/28592069/207568737-3eed0f16-9b6d-46ee-ab8a-212961ca8719.png">
<img width="776" alt="2022-12-14 11_11_44-" src="https://user-images.githubusercontent.com/28592069/207568742-e99921e7-5f58-45a6-b3e9-e19556a0957b.png">

**Comment by melohagan** [comment, marked as answer]
Raised a bug here! https://github.com/Budibase/budibase/issues/9142
